components/Footer/src/index.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import BaseProps from '@gemeente-denhaag/baseprops';
+import ResponsiveContent from '/components/ResponsiveContent';
+
+import './index.scss';
+
+export interface FooterListItemData {
+  label: string;
+  url: string;
+}
+
+export interface FooterLegalProps extends Omit<BaseProps, 'tabIndex' | 'classNamees' | 'children'> {
+  legalData?: Array<FooterListItemData>;
+  copyrightLabel?: string;
+}
+
+export interface FooterSocialLinks {
+  icon: string;
+  title: string;
+  url: string;
+}
+
+export interface FooterSocialData {
+  links: Array<FooterSocialLinks>;
+  title: string;
+}
+
+export interface NewsLetterData {
+  title: string;
+  text: string;
+  buttonText: string;
+  url: string;
+}
+
+export interface ContactData {
+  title: string;
+  links: Array<FooterListItemData>;
+}
+
+export interface FooterContactProps extends Omit<BaseProps, 'tabIndex' | 'classNamees' | 'children'> {
+  socialData?: FooterSocialData;
+  newsletterData?: NewsLetterData;
+  contactData?: ContactData;
+}
+
+export interface FooterProps extends Omit<BaseProps, 'tabIndex' | 'classNamees' | 'children'> {
+  legalData?: Array<FooterListItemData>;
+  copyrightLabel?: string;
+  socialData?: FooterSocialData;
+  newsletterData?: NewsLetterData;
+  contactData?: ContactData;
+}
+
+export const FooterLegal: React.FC<FooterLegalProps> = (props: FooterLegalProps) => {
+  return (
+    <section className="denhaag-footer__legal">
+      <div className="denhaag-responsive-content">
```

Comment:
Not sure but maybe we could combine `denhaag-resonsive-content` with `denhaag-footer__legal` or `denhaag-footer-legal__container`?